src/AutoRest.CSharp/Common/Output/Models/Types/SchemaObjectType.cs

```diff
@@ -78,7 +78,7 @@ public SchemaObjectType(ObjectSchema objectSchema, BuildContext context) : base(
                 SourceMemberMapping? memberMapping = _sourceTypeMapping?.GetForMember("$AdditionalProperties");
```

Comment:
We should use the real property name here (`AdditionalProperties`)

Reply:
^^^

>Hey @pakrym - Not sure what you meant by your "might need to change it, we have a real property" comment on the commit.

This.

Reply:
Discussed in chat and we decided against this.